Libraries/Image/RCTImageView.m

```diff
@@ -44,6 +44,8 @@ @interface RCTImageView ()
 @property (nonatomic, copy) RCTDirectEventBlock onLoad;
 @property (nonatomic, copy) RCTDirectEventBlock onLoadEnd;
 
+@property (nonatomic, assign, getter=isDisplayingSource) BOOL displayingSource;
```

Comment:
Can you explain the rules for this determination?  This file is using a mixture of properties and ivars.  I didn't see anything about how these determinations are made in the guide on code contribution.

This file uses properties and ivars.  Additionally, it accesses some of the properties via the ivar, so I confused on your best practices here.